src/back/Update/TorrentsClients.php

```diff
@@ -216,11 +216,18 @@ private function updateUntracked(bool $doUpdateUntracked): void
 
                 if ($countUntracked > 150) {
                     $this->logger->notice(
-                        'Хранится много сторонних раздач. Рекомендуется отключить поиск сторонних раздач или добавить подразделы в хранимые.'
+                        'Хранится много сторонних раздач ({count} шт.). Рекомендуется отключить поиск сторонних раздач или добавить подразделы в хранимые.',
+                        ['count' => $countUntracked]
```

Comment:
What is the point of this amendment if the number of topics is displayed literally one line higher?
```
12.05.2025 15:08:06 INFO: Найдено уникальных сторонних раздач в клиентах: 8750 шт.
12.05.2025 15:08:06 NOTICE: Хранится много сторонних раздач. Рекомендуется отключить поиск сторонних раздач или добавить подразделы в хранимые.
```

Reply:
It was not obvious of what is considered "many", the line above mentioned only about the "unique" ones found lately, and it was not clear whether the "many" number considered those not unique; it might seem that the total stored number is higher than the newly found unique ones. These two log lines were perplexing before I added the number.